package.json

```diff
@@ -48,7 +48,7 @@
     "trash": "6.1.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.1000.0",
+    "@angular-devkit/build-angular": "^0.1100.0",
```

Comment:
I don't mind too much, but it's generally better not to change versions and keep a PR focused on one thing. 

We could update to Angular 11 in a separate PR 🤝